dev/io.openliberty.microprofile.telemetry.1.0.internal_fat/fat/src/io/openliberty/microprofile/telemetry/internal_fat/common/TestSpans.java

```diff
@@ -78,7 +78,7 @@ public static void assertLinearParentage(List<SpanData> spans) {
      * @throws Exception if {@code runnable} throws an exception
      */
     public Span withTestSpan(ThrowingRunnable runnable) {
-        String spanName = "testSpan-" + request.getRequestURI();
+        String spanName = "testSpan-" + request.getRequestURL();
```

Comment:
Why this change?

Reply:
Can't see an obvious reason for this, I'll change it back and check nothing breaks.